packages/forklift-console-plugin/src/modules/Plans/utils/helpers/getPlanPhase.ts

```diff
@@ -117,5 +117,11 @@ export const isPlanEditable = (plan: V1beta1Plan) => {
   );
 };
 
+export const isPlanArchived = (plan: V1beta1Plan) => {
+  const planStatus = getPlanPhase({ obj: plan });
+
+  return planStatus === 'Archiving' || planStatus === 'Archived';
```

Comment:
Archiving and Archived will probably be widely used, therefore it better to add them to a const file and prevent future typos

Reply:
Agree but current code is using the  [PlanPhase](https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/forklift-console-plugin/src/modules/Plans/utils/types/PlanPhase.ts) type based on strings and this type is used all over by comparing to strings.
Not complicated to refactor, but better do it in a follow up PR as well since touching other files.

Reply:
PlanPhase can be an enum and used in the compare planStatus === PlanPhase.Archiving etc... lets refactor it

Reply:
@metalice 
Sure!
But I really prefer not to do it as part of this PR, but rather solve it today in a follow up one, since:
1. it is not related to this fix and touches other components/pages so it's better to separate.
2. It will generate unnecessary code conflicts since there is a merged code from me which is related to the plan status, e.g. https://github.com/kubev2v/forklift-console-plugin/blob/3a656a550c0e941d7f8b33e7073f17ed86c4740b/packages/forklift-console-plugin/src/modules/Plans/utils/helpers/getPlanPhase.ts#L120-L124

Let's merge the opened PRs relevant for plan status and I'll handle this  issue separately.

Reply:
A follow up PR: https://github.com/kubev2v/forklift-console-plugin/pull/1408